docs/howtos/other/certificate_issuers.md

```diff
@@ -6,21 +6,22 @@ title: ""
 
 # Using a Certificate Issuer
 
-Epinio comes with multiple [cert-manager cluster issuers](https://cert-manager.io/docs/configuration/) for creating certificates:
+Epinio comes with multiple [cert-manager ClusterIssuer](https://cert-manager.io/docs/configuration/) for creating certificates.  
+By default it installs the `epinio-ca` and the `selfsigned-issuer`. If specified it will install also the `letsencrypt-staging` or `letsencrypt-production` issuer, to create certificates issued from [Let's Encrypt](https://letsencrypt.org/).
```

Comment:
```suggestion
By default it installs the issuers `epinio-ca` and `selfsigned-issuer`.
If specified it will also install either `letsencrypt-staging` or `letsencrypt-production` , to create certificates issued from [Let's Encrypt](https://letsencrypt.org/).
```